2015/proposals/banaszkiewicz-piotr-amy.md

```diff
@@ -0,0 +1,214 @@
+# Enhance Amy, a workshop-management platform for Software Carpentry
+
+## Abstract
+
+The number of Software Carpentry's workshops run weekly dynamically grows. All
```

Comment:
On Tue, Mar 24, 2015 at 11:38:20AM -0700, Piotr Banaszkiewicz wrote:
> > +The number of Software Carpentry's workshops run weekly
> >  dynamically grows. All
> 
> I was unable to find numbers so I inquired @gvwilson about them.

So I'm not sure about workshops, but I think this is the last graph
showing the increase in student numbers [1].

[1]: http://software-carpentry.org//blog/2015/01/announcing-2015-steering-committee.html